README.md

```diff
@@ -27,16 +27,10 @@ A third party security audit was performed by Cure53, you can see the full repor
 
 ## Building
 
-`runc` only supports Linux. It must be built with Go version 1.21 or higher.
+`runc` only supports Linux. It must be built with Go version 1.22.4 or higher.
```

Comment:
Now that go enforces the go version in go.mod, don't we want to just delete the last sentence here?

Reply:
I think it's fine as it is.